### PR TITLE
Requirements.txt and some minor changes

### PR DESCRIPTION
**Pull request by slavong**
When I cloned this repo I had some issues ansd these changes should make it easier to get this repo up and running.

### CODE REVIEW

File: benchmark.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
```
Comment (sohaibiftikhar) [inline benchmark.py]: I would do `#!/usr/bin/env python3` here just to be safe.

File: README.md [hunk 2]
```diff
@@ -5,15 +5,27 @@ Implements the [TPCH benchmark](http://www.tpc.org/tpch/) for Postgres
 * The benchmark requires TPC-H dbgen:
 
 ```
-wget https://github.com/electrum/tpch-dbgen/archive/32f1c1b92d1664dba542e927d23d86ffa57aa253.zip
-unzip 32f1c1b92d1664dba542e927d23d86ffa57aa253.zip
-mv tpch-dbgen-32f1c1b92d1664dba542e927d23d86ffa57aa253 tpch-dbgen
-rm 32f1c1b92d1664dba542e927d23d86ffa57aa253.zip
+wget -q https://github.com/electrum/tpch-dbgen/archive/32f1c1b92d1664dba542e927d23d86ffa57aa253.zip -O tpch-dbgen.zip
+unzip tpch-dbgen.zip -d tpch-dbgen/ && rm tpch-dbgen.zip
```
Comment (sohaibiftikhar) [inline README.md]: Did you test this? I think this might create two levels of directories?

Comment (slavong) [reply README.md]: You are right. Let me fix this.

File: benchmark.py (hunk 1 above)
Comment (slavong) [reply benchmark.py]: fine with me.

Comment (sohaibiftikhar) [reply benchmark.py]: @slavong Could you also please do this so it can be merged.

**Comment by slavong** [comment]
That should be it now. Sorry for the delay, but I was busy.

**Pull request merged**